src/htmx.js

```diff
@@ -44,7 +44,8 @@ return (function () {
                 settlingClass:'htmx-settling',
                 swappingClass:'htmx-swapping',
                 allowEval:true,
-                attributesToSettle:["class", "style", "width", "height"]
+                attributesToSettle:["class", "style", "width", "height"],
+                wsReconnectInterval: 10000
```

Comment:
10 seconds seems like a long time, but I don't do any websocket programming, so I'm willing to defer to expertise here.  Just want to make sure this wasn't an accidental value.  :)

Reply:
I actually put some more thoughts into it. Going to push a new version soon.

I though about the use-cases and potential users of htmx. From my perspective, users would just expect the most up-to-date reconnection algorithm or would want to provide their own. So, I removed the number completely and added a more sophisticated full-jitter version instead of a constant. Additionally, I added ability to specify an own implementation.

Reply:
A well-described analysis can be found from AWS: https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/